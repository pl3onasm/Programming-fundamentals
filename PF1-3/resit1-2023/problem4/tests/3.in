Logicians state that they will accept that 1+1=3 according axiomatic identities that say that it is true that 1=2.
