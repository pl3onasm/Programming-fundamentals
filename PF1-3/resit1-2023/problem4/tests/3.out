they say it is true
